src/rustdoc_schema.graphql

```diff
@@ -499,12 +499,18 @@ type Method implements Item & FunctionLike {
 A specific attribute applied to an Item.
 """
 type Attribute {
-  """
-  The textual representation of the attribute.
+  as_string: String!
+  is_inner: Boolean!
+  content: AttributeValue!
+}
 
-  For example: "#[repr(C)]"
-  """
-  value: String!
+type AttributeValue {
+  base: String!  # the path, e.g. "cfg_attr" or "must_use"
+  as_string: String!  # the entire value represented as a string
```

Comment:
Two suggestions on this:
- Let's rename `as_string` to something like `full_value` or `raw_value`. Right now, it's confusing because all the other properties are strings too, so `as_string` doesn't differentiate itself.
- Let's move `as_string` to the top of the property list. It's the simplest-to-understand property, and readers should see it first before moving to anything more sophisticated.

In general, you might notice a theme here: when we have extra degrees of freedom, let's use them in a way that optimizes for the downstream user's experience. For example, the schema parser doesn't care about property order (i.e. it's a degree of freedom) so let's lay out the properties in an order that will help the humans that read this file.

Reply:
Fully agree! I like `raw_value` more than `full_value`, I think it better indicates that it's the value before any parsing.